stock_secondary_unit/views/stock_picking_views.xml

```diff
@@ -6,15 +6,15 @@
         <field name="name">Stock Picking Secondary Unit</field>
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="stock.view_picking_form" />
-        <field name="groups_id" eval="[(4, ref('uom.group_uom'))]" />
         <field name="arch" type="xml">
             <xpath
                 expr="//field[@name='move_ids_without_package']/tree/field[@name='product_uom_qty']"
                 position="before"
             >
                 <field
                     name="secondary_uom_qty"
-                    attrs="{'column_invisible': [('parent.immediate_transfer', '=', True)], 'readonly': ['|', ('is_initial_demand_editable', '=', False), '&amp;', '&amp;', ('show_operations', '=', True), ('is_locked', '=', True), ('is_initial_demand_editable', '=', False)]}"
+                    groups="uom.group_uom"
+                    readonly="not is_initial_demand_editable"
```

Comment:
Wouldn't the column_invisible attribute and the rest of the readonly conditions be missing?